services/gcp/cloudsql/instances.yml

```diff
@@ -0,0 +1,163 @@
+    notes: >-
+      A common use case of tag bindings is for use in IAM policy conditions. If the user has any
+      policies that use tag bindings to enforce conditions, creating a tag on a resource allows them
+      to escalate their access to that resource. Also requires getIamPolicy or knowledge of the IAM
+      policy from some other means.
+  delete:
+    risks: [destruction:data, destruction:infra]
+    notes: >-
+      With this privilege, users can delete Cloud SQL instances. It poses the risks of service disruption, permanent data loss, and infrastructure damage.
+  deleteTagBinding:
+    risks: [escalation:privilege, destruction:policy, impact:access]
+    notes: >-
+      A common use case of tag bindings is for use in IAM policy conditions. The "destruction:policy" and
+      "impact:access" risks apply if the tag is used in any policies.
+  demoteMaster:
+    risks: [impact:dos]
+    notes: >-
+      This permission allows converting an instance to a read replica. Since read replicas do not allow write requests, this
+      can cause a denial of service if the instance is handling write requests.
+    links:
+      - https://cloud.google.com/sql/docs/mysql/replication
+  export:
+    risks: [exfiltration:data]
+  failover:
+    risks: [impact:dos]
+    notes: >-
+      This causes Cloud SQL to switch serving data from a secondary instance instead of the primary.
+      Google expects the failover operation to render the instance inaccessible for about 60 seconds, so this could 
+      produce a DOS if executed repeatedly.
+  get:
+    risks: [discovery:infra, discovery:policy]
+    notes: >-
+      Allows access to instance metadata and settings, including IP addresses, authorized networks, and backup settings.
+  getDiskShrinkConfig:
+    risks: []
+    notes: >-
+      The specific use of this permission is unknown.
+  import:
+    risks: [impact:consumption, impact:dos, impact:manipulation]
+    notes: >-
+      Allows importing data to an instance. If enough data is imported to exceed the disk space of the instance, 
+      will cause a DOS until the instance is manually resized.
+  list:
+    risks: [discovery:infra, discovery:policy]
+    notes: >-
+      Allows access to instance metadata and settings, including IP addresses, authorized networks, and backup settings.
+  listEffectiveTags:
+    risks: [discovery:policy]
+  listServerCas:
+    risks: []
+  listTagBindings:
+    risks: [discovery:policy]
+  login:
+    risks: []
+    notes: >-
+      The specific use of this permission is unknown.
+  migrate:
+    risks: []
+    notes: >-
+      The specific use of this permission is unknown.
+  performDiskShrink:
+    risks: []
+    notes: >-
+      The specific use of this permission is unknown.
+  promoteReplica:
+    risks: [impact:spend]
+    notes: >-
+      This permission allows converting a read replica to a primary instance.
+    links:
+      - https://cloud.google.com/sql/docs/mysql/replication
+  reencrypt:
```

Comment:
How do I get spell check to ignore this? It's the name of the permission.

Reply:
You add this to the cspell allow list